packages/full-stack-tests/src/components/unified-selection/Table.test.tsx

```diff
@@ -123,7 +120,7 @@ describe("Learning snippets", async () => {
         // __PUBLISH_EXTRACT_START__ Presentation.Components.UnifiedSelection.TableWithinUnifiedSelectionContext
         <UnifiedSelectionContextProvider imodel={imodel}>
           <MyTable imodel={imodel} />
-        </UnifiedSelectionContextProvider>
+        </UnifiedSelectionContextProvider>,
```

Comment:
I generally don't mind the ending comma, but in this situation it's an example where I'd like to avoid it. 

It's fine if the solution is ugly since the snippet extraction situation is exceptional, so anything like this works too, if it can't be solved more elegantly:
```ts
const { container } = render(
  (
    // __PUBLISH_EXTRACT_START__ Presentation.Components.UnifiedSelection.TableWithinUnifiedSelectionContext
    <UnifiedSelectionContextProvider imodel={imodel}>
      <MyTable imodel={imodel} />
    </UnifiedSelectionContextProvider>
    // __PUBLISH_EXTRACT_END__
  ),
);
```

Reply:
Moved example from `render` function to test component so there is no comma at the end